ilastik/workflows/objectClassification/objectClassificationWorkflow.py

```diff
@@ -551,17 +556,64 @@ class ObjectClassificationWorkflowPixel(ObjectClassificationWorkflow):
     workflowName = "Object Classification (from pixel classification)"
     workflowDisplayName = "Pixel Classification + Object Classification"
 
-    def setupInputs(self):
-        data_instructions = 'Use the "Raw Data" tab on the right to load your intensity image(s).'
-        
-        self.dataSelectionApplet = DataSelectionApplet( self, 
-                                                        "Input Data", 
-                                                        "Input Data", 
-                                                        batchDataGui=False,
-                                                        forceAxisOrder=None, 
-                                                        instructionText=data_instructions )
-        opData = self.dataSelectionApplet.topLevelOperator
-        opData.DatasetRoles.setValue(['Raw Data'])
+    @property
+    def ExportNames(self):
+        class ExtraExportNames(SlotNameEnum):
+            PIXEL_PROBABILITIES = super(self.__class__, self).ExportNames.getNext()
```

Comment:
This reorders the "tabs" in the input data applet gui. With "optional" roles we could probably automatically order those in a meaningful way.

Reply:
Oh well... falling back to straighforward IntEnums which dictate the order of the tabs -.-